cli/cfncluster/cli.py

```diff
@@ -21,13 +21,25 @@
 from . import cfncluster
 from . import easyconfig
 
+#
+# configuration
+#
+username_map = { 'alinux' : 'ec2-user',
```

Comment:
You don't need this map here, you aren't using it anyway.

Reply:
Removed. Actually I can get rid of the entire dict by using `get_template()` and looking at the mappings section: 

```
"Mappings" : {
    "AWSRegionOS2AMI" : {
     ...
    },
    "OSFeatures" : {
      "centos6" : {
        "User" : "centos",
        "RootDevice" : "/dev/sda1"
      },
      "centos7" : {
        "User" : "centos",
        "RootDevice" : "/dev/sda1"
      },
      "alinux" : {
        "User" : "ec2-user",
        "RootDevice" : "/dev/xvda"
      },
      "ubuntu1404" : {
        "User" : "ubuntu",
        "RootDevice" : "/dev/sda1"
      },
      "ubuntu1604" : {
        "User" : "ubuntu",
        "RootDevice" : "/dev/sda1"
      }
    },
```